osgp/shared/osgp-ws-distributionautomation/src/main/java/org/opensmartgridplatform/ws/da/config/DistributionAutomationWebServiceConfig.java

```diff
@@ -32,7 +32,8 @@ public PayloadValidatingInterceptor payloadValidatingInterceptor() {
         final Resource[] resources = new Resource[] { new ClassPathResource(COMMON_XSD_PATH),
                 new ClassPathResource(DISTRIBUTION_AUTOMATION_XSD_PATH) };
         payloadValidatingInterceptor.setSchemas(resources);
-        payloadValidatingInterceptor.setValidateRequest(true);
+        // TODO: with .setValidateRequest(true) the endpoint errors on receiving a request
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1135)

Reply:
I logged a bug for this situation, see: https://smartsocietyservices.atlassian.net/browse/DA-374